clustered/client/src/test/java/org/ehcache/clustered/client/internal/UnitTestConnectionService.java

```diff
@@ -240,6 +246,8 @@ public static PassthroughServer remove(URI uri) {
           LOGGER.error("Entity destroy failed: ", ex);
         } catch (EntityNotFoundException ex) {
           LOGGER.error("Entity destroy failed: ", ex);
+        } catch (PermanentEntityException e) {
+          LOGGER.error("Entity destroy failed (permanent???): ", e);
```

Comment:
Yes, since that is in test code, attracting attention to that point. A permanent entity is a special kind of entity that has no configuration and the lifecycle of the server.